gradle.properties

```diff
@@ -8,22 +8,24 @@ pluginVersion = 0.3.12
 
 # See https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
 # for insight into build numbers and IntelliJ Platform versions.
-pluginSinceBuild = 213.5744.223
-pluginUntilBuild = 221.*
+pluginSinceBuild = 222.3345.99
+pluginUntilBuild = 222.*
 
 # IntelliJ Platform Properties -> https://github.com/JetBrains/gradle-intellij-plugin#intellij-platform-properties
 platformType = PC
-platformVersion = 2021.3.2
+platformVersion = 222.3345-EAP-CANDIDATE-SNAPSHOT
 
 # Plugin Dependencies -> https://plugins.jetbrains.com/docs/intellij/plugin-dependencies.html
 # Example: platformPlugins = com.intellij.java, com.jetbrains.php:203.4449.22
 platformPlugins = python-ce
 
 # Java language level used to compile sources and to generate the files for - Java 11 is required since 2020.3
 javaVersion = 11
+languageVersion = 1.7
+apiVersion = 1.6
```

Comment:
If we are targeting 222 as a minimum, then we should be using api 1.7 and java 11, I think we should a toolchain here like in the plugin template project

Reply:
https://github.com/JetBrains/intellij-platform-plugin-template/blob/f3250aeef4c68761bc7d9dbdd59396cf7260efe3/build.gradle.kts#L31